### PR TITLE
Boyscout/chevron icon

### PR DESCRIPTION
**Pull request by wakooka**
## Description

Added up prop on ChevronIcon

## How it was tested

Storybook

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Unreleased
 
-[...]
+- **[UPDATE]** Added `up` prop on `ChevronIcon`
+
+# v49.0.0 (25/02/2021)
```
Comment (mcellier) [inline CHANGELOG.md:5]: Weird changelog. Merge issue?

File: src/icon/chevronIcon.tsx [hunk 2]
```diff
@@ -6,9 +6,10 @@ export type ChevronIconProps = Icon &
   Readonly<{
     down?: boolean
     left?: boolean
+    up?: boolean
   }>
 
-export const ChevronIcon = ({ down, left, ...props }: ChevronIconProps) => (
+export const ChevronIcon = ({ down, left, up, ...props }: ChevronIconProps) => (
```
Comment (mcellier) [inline src/icon/chevronIcon.tsx:12]: nit: We could also introduced a `Direction` enum to handle this. We'll have trouble if we pass multiple props, i.e. `<ChevronIcon down left/>`

**Review by mcellier — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (wakooka) [reply CHANGELOG.md:5]: Fixed!

File: src/icon/chevronIcon.tsx (hunk 2 above)
Comment (wakooka) [reply src/icon/chevronIcon.tsx:12]: Done

**Pull request merged**